doc/readme.txt

```diff
@@ -350,14 +350,15 @@ BYTES NAME - brief description
 
 			4. ACKNOWLEDGEMENTS
 
-The AXAF Guide and Acquisition Star Catalog version 1.6 was prepared
-from AGASC1.5 primarily by Brett Unks, Tom Aldcroft, and Rob Cameron, using code
-written by Brett Unks.  Thanks to the entire Star Selection and Aspect
-Working Group for its input in the development and testing of this
+The AXAF Guide and Acquisition Star Catalog version 1.7 was prepared
+from AGASC1.6 and AAVSO (American Association of Variable Star Observers)
+Photometric All-Sky Survey (APASS) DR9, primarily by Malgosia Sobolewska,
+Tom Aldcroft, and Jean Connelly. Thanks to the entire Star Selection and
+Aspect Working Group for its input in the development and testing of this
 catalog. The Chandra X-ray Center is supported through NASA Contract
-NAS8-39073. Information about Chandra and the Chandra X-ray
-Observatory Center may be found on the WWW at http://chandra.harvard.edu/ 
-Detailed information about the catalog and its construction
-can be obtained from the Chandra aspect web page at
-http://asc.harvard.edu/mta/ASPECT/ or by emailing:
+NAS8-39073. Information about Chandra and the Chandra X-ray Observatory
+Center may be found on the WWW at http://chandra.harvard.edu/ 
+Detailed information about the catalog and its construction can be
+obtained from the Chandra aspect web page at
+https://cxc.harvard.edu/mta/ASPECT/agasc1p7 or by emailing:
```

Comment:
I updated the URLS here to and in the other file to go directly to the agasc1p7 page.  Any objections @malgosias ?

Reply:
I suppose the agasc link isn't really *the* Chandra aspect web page, but it is the Chandra aspect web page relevant for this product.